packages/@icon-magic/icon-models/src/interface.ts

```diff
@@ -91,6 +91,13 @@ export interface DistributeConfig {
   svg?: SVGOptions;
 }
 
+/**
+ * An object containing any meta information about the icon
+ */
+export interface MetaData {
+  [key: string]: any;
```

Comment:
related to my comment on tim's issue, seems like this interface should at least define `nameSizeMapping` as optional — it's used in a few places

Reply:
Ok.